src/utils/ledger.js

```diff
@@ -71,12 +89,10 @@ if (IPC_RENDERER) {
       // first assemble data to be sent
       const arr = [];
       if (changeIndex > -1) {
-        // With change output
-        arr.push(hathorLib.transaction.intToBytes(1, 1));
-        // Change index of array
-        arr.push(hathorLib.transaction.intToBytes(changeIndex, 1));
+        // encode the bit indicating existence of change and change index on first byte
+        arr.push(hathorLib.transaction.intToBytes(0x80 | changeIndex, 1))
```

Comment:
You use the first bit to indicate the existence of change output and then the other 7 bits you identify what's the index?

If that's the case, we can have only 128 outputs, right?

Reply:
Thanks, this is wrong. I tested this case on a script so the ledger parses ok, but i didn't test the wallet on this. It should be 1 byte for `has_change_output + len(change_bip32_path)` another byte for index and then the actual path. since the path length can be at most 5, 7 bits is more than enough.